main.go

```diff
@@ -45,8 +59,22 @@ func main() {
 	lambda.Start(Handler)
 }
 
-func sendMessage(document string) error {
-	message := fmt.Sprintf("{\"action\": \"touch\", \"body\": %s}", document)
+func uniqueSubjects(in []sparql.Triple) []string {
+	u := make([]string, 0, len(in))
+	m := make(map[string]bool)
```

Comment:
We're using the `map` here exclusively to determine if the `u` slice has any duplicate strings? I assume that's a golang convention because the slice doesn't provide a more efficient way to check this (that doesn't involve, e.g., looping over the whole slice again and again)?

Reply:
@mjgiarlo Correct, as far as I could fine, this was really the only way to remove duplicates (or determine duplicates) within a given slice.

Reply:
OK. Just checking -- this is fine. :)